src/libs/GrowlNotification/index.js

```diff
@@ -0,0 +1,89 @@
+import React, {Component} from 'react';
+import {
+    Text, View, Animated,
+} from 'react-native';
+import {
+    Directions, FlingGestureHandler, State, TouchableWithoutFeedback,
+} from 'react-native-gesture-handler';
+import colors from '../../styles/colors';
+import Icon from '../../components/Icon';
+import {Checkmark, Exclamation} from '../../components/Icon/Expensicons';
+import styles from '../../styles/styles';
+import GrowlNotificationContainer from './GrowlNotificationContainer';
+
+const types = {
+    success: {
+        icon: Checkmark,
+        iconColor: colors.green,
+    },
+    error: {
+        icon: Exclamation,
+        iconColor: colors.red,
+    },
+    warning: {
+        icon: Exclamation,
+        iconColor: colors.yellow,
+    },
+};
+
+const outDistance = -255;
```

Comment:
file-level constants should be in `UPPER_SNAKE_CASE`

```suggestion
const OUT_DISTANCE = -255;
```

Also, perhaps we can come up with a more descriptive name for this constant that makes it more immediately clear what it is.

Reply:
Changed